src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitTextField.razor

```diff
@@ -35,6 +36,7 @@
                @onfocusout="@HandleFocusOut"
                @onfocus="@HandleFocus"
                @onchange="@HandleChange"
+               @oninput="@HandleChange"
```

Comment:
There is no need to use both "onchange" and "oninput" events together.
please use only the "oninput" event.

Reply:
done